active/PagerControl/PagerControl.md

```diff
@@ -0,0 +1,405 @@
+
+**Important APIs:** [PagerControl class](https://docs.microsoft.com/en-us/uwp/api/microsoft.ui.xaml.controls.pagercontrol)
+
+This control can be configured to use one of three different display modes shown below:
+
+### ComboBox Mode
+![](images/pager_control_combo.jpg)
+
+### NumberBox Mode
+![](images/pager_control_number.jpg)
+
+### NumericalButtonPanel Mode 
+![](images/pager_control_button.jpg)
+
+## Is this the right control? 
+
+Use a **PagerControl** to create a navigation interface for an end user to page through content instead of scrolling or panning. 
+
+Scrolling through all content in a single view can make it harder for the end user to find specific information and it can lead to performance issues when the content needs to be refreshed. **PagerControl** can be used as an alternate solution so the user can consume a subset of the content without needing to scroll. They can then page through the rest of the content to find what they are looking for. 
+
+## Examples
+
+A pager control has three different display modes and each display mode has configurable components within them. 
+
+### ComboBox Display Mode
+
+![](images/pager-control-combobox.png)
+
+### Pager Control Editable NumberBox Display Mode 
+
+![](images/pager-control-editable-textbox.png)
+
+### Pager Control Numerical Button Panel Display Mode
+
+![](images/pager-control-numerical-button-panel.png)
+
+A pager control does not require you to provide a max number of pages to the control. If the max number of pages is unknown, here is what each display mode will look like. 
+
+### ComboBox Display Mode
+
+(add comps)
+
+### Pager Control Editable NumberBox Display Mode 
+
+(add comps)
+
+### Pager Control Numerical Button Panel Display Mode
+
+(add comps)
+
+You can choose to customize the look of a pager control by changing the visibility of the first, previous, next, and last buttons. They can be configured to always be visible, hidden, or hidden when the selected index is the first or last page in the range. When the last page is selected, the next and last buttons will be disabled and same when the first page is selected for the first and previous page. The prefix and suffix text can also be customized to display a string you provide. 
+
+## Creating a pager control 
+
+Here is the XAML for how to add a pager control to your application using the auto display mode. Auto will choose to use the combo box mode if the NumberOfPages property is less than 11 and will show the number box mode if it is greater than 11. In this example, combo box will be chosen. 
+
+XAML
+```XAML
+<Grid>
+        <controls:DataGrid x:Name="dataGrid" 
+                Height="600" Margin="12"
+                AutoGenerateColumns="True"
+                ItemsSource="{x:Bind MyViewModel.Items}" />  
+        <muxc:PagerControl x:Name="mainPagerControl"
```

Comment:
Is it intentional that both the DataGrid and the Pager are in the same row ? We might want to show two rows and have the PagerControl be in the second row.

Reply:
the pager is grid.row = 2 and data grid is grid.row =1. Is there another way for me to make sure they are not in the same row?

Reply:
? Neither one of these is setting Grid.Row

Reply:
@MikeHillberg is that something that happens with the git commits? It has happened to me before with ProgressRing and I'm not sure how to fix it.

Reply:
It's conspiring against you :) I would just re-copy it, maybe there was a merge error.